torch/csrc/jit/python/init.cpp

```diff
@@ -1386,7 +1402,7 @@ void initJITBindings(PyObject* module) {
               "__radd__",
               [](c10::SymIntNode a, py::object b) -> c10::SymIntNode {
                 auto snb = toSymIntNode(a, b);
-                return a->add(snb);
+                return snb->add(a);
```

Comment:
what's going on here?

Reply:
Bugfix but it actually doesn't matter cuz add is commutative